CMakeLists.txt

```diff
@@ -77,6 +78,7 @@ endif()
 # options
 option(CYGWIN_USE_POSIX "Should the POSIX API be used for cygwin. Ignored if the system isn't cygwin." OFF)
 option(ENABLE_CXX11 "Should the c++11 parts (srt-live-transmit) be enabled" ON)
+option(DISABLE_SUPPORT_APPLICATIONS "Should the Support Applications be Disabled?" OFF)
```

Comment:
In this system there's an enable/disable symmetry in `configure`, therefore all boolean enabler options must have `ENABLE_` prefix. Also, for options please use as short name as possible - `ENABLE_APPS` should be enough.

Reply:
`ENABLE_APPS` or `ENABLE_SAMPLE_APPS`

Reply:
Other parts of the code refers to them as support applications. Maybe there needs to be some consistency about what to call them. Either Sample Applications or Support Applications.

Reply:
Committed a change to use ENABLE_APPS